packages/templates/src/raw/all-about-cars/template.json

```
@@ -0,0 +1,7242 @@
+              [0, 300],
+              [1, 300],
+              [0, 400],
+              [1, 400],
+              [0, 600],
+              [1, 600],
+              [0, 700],
+              [1, 700],
+              [0, 900]
+            ],
+            "metrics": {
+              "upm": 1000,
+              "asc": 1133,
+              "des": -388,
+              "tAsc": 1133,
+              "tDes": -388,
+              "tLGap": 0,
+              "wAsc": 1133,
+              "wDes": 388,
+              "xH": 500,
+              "capH": 692,
+              "yMin": -285,
+              "yMax": 1082,
+              "hAsc": 1133,
+              "hDes": -388,
+              "lGap": 0
+            }
+          },
+          "fontSize": 40,
+          "backgroundColor": {
+            "color": {
+              "r": 196,
+              "g": 196,
+              "b": 196
+            }
+          },
+          "lineHeight": 1,
+          "textAlign": "center",
+          "padding": {
+            "locked": true,
+            "hasHiddenPadding": false
+          },
+          "type": "text",
+          "content": "<span style=\"color: #002e73\">m</span>",
+          "fontWeight": 400,
+          "borderRadius": {
+            "locked": true,
+            "topLeft": 2,
+            "topRight": 2,
+            "bottomRight": 2,
+            "bottomLeft": 2
+          },
+          "width": 55,
+          "height": 61,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "basedOn": "4b7a6c25-b2df-40b0-9a2d-4875b9d32b77",
+          "id": "0598f042-d8b3-44b9-933c-fcc10e03edca",
+          "x": 255,
+          "y": 62
+        },
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "type": "sticker",
+          "width": 495,
+          "height": 494,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "sticker": {
+            "type": "selfDrivingCarsMultipleBlueRings"
+          },
+          "basedOn": "045aa174-4326-485e-ab49-061d5d7f517c",
+          "id": "0be7ded0-d3d7-46d4-a0c2-e48f00b51273",
+          "x": -50,
+          "y": 56
+        },
+        {
+          "opacity": 100,
+          "flip": {
+            "vertical": false,
+            "horizontal": false
+          },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "resource": {
+            "type": "image",
+            "mimeType": "image/png",
+            "creationDate": "2021-07-22T09:49:57",
+            "src": "__WEB_STORIES_TEMPLATE_BASE_URL__/images/templates/all-about-cars/page3_4_5_11_12_13_img1.png",
+            "width": 468,
+            "height": 444,
+            "id": 0,
+            "alt": "Selfdriving Smart Car High-Res Vector Graphic",
```

```suggestion
            "alt": "Selfdriving Smart Car",
```

Same for the other instances of this string.